.github/workflows/ci-format.yml

```diff
@@ -1,12 +1,6 @@
 name: SMARTS CI Format
 
-on:
-  push:
-    branches-ignore:
-      - master
-  pull_request:
-    branches-ignore:
-      - master
+on: [push, pull_request]
```

Comment:
We have merged `master` back into `develop` in cases so I think this might trigger an excess run on master when opening a pull request for that.